modules/EventTracker.js

```diff
@@ -0,0 +1,187 @@
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define(['underscore', 'backbone', './ServiceCell'], factory);
+  } else if (typeof exports === 'object') {
+    module.exports = factory(require('underscore'), require('backbone'), require('./ServiceCell'));
+  } else {
+    root.Torso = root.Torso || {};
+    root.Torso.EventTracker = factory(root._, root.Backbone, root.Torso.ServiceCell);
+  }
+}(this, function(_, Backbone, ServiceCell) {
+  'use strict';
+
+  var EventTracker = ServiceCell.extend({ 
```

Comment:
You created all these wonderful triggered events inside Model, Router, and View, but I can't see you listening to any of them to generate events from this class? That begs the question, how are you planning to listen to each model, view, and router in order to generate the events? As long as you only have one EventTracker and it's created by default, it's possible to have in the constructors of the Model, View, and Router to add an "on" that registers the events to the EventTracker.

In Torso, we have not made a module rely on the existence of an instance of another module, which is what the previous suggestion requires. Do you have thoughts on how to add the dependency on an instance of an EventTracker but still have EventTracker be optional?